modules/programs/broot.nix

```
@@ -43,6 +44,14 @@ in {
       '';
     };
 
+    modal = mkOption {
+      default = false;
+      type = types.bool;
+      description = ''
+        Whether to enable modal (vim) mode.
+      '';
+    };
+
```

I think this could be a `mkEnableOption`:

```suggestion
    modal = mkEnableOption "modal (vim) mode";
```

---

Thanks for the suggestion!